code/controllers/subsystems/ticker.dm

```diff
@@ -0,0 +1,564 @@
+
+	var/hide_mode = FALSE 				// If the true game mode should be hidden (because we chose "secret")
+	var/datum/game_mode/mode = null		// The actual gamemode, if selected.
+
+	var/end_game_state = END_GAME_NOT_OVER	// Track where we are ending game/round
+	var/restart_timeleft				// Time remaining until restart in desiseconds
+	var/last_restart_notify				// world.time of last restart warning.
+	var/delay_end = FALSE               // If set, the round will not restart on its own.
+
+	var/login_music						// music played in pregame lobby
+
+	var/list/datum/mind/minds = list()	// The people in the game. Used for objective tracking.
+
+	// TODO - I am sure there is a better place these can go.
+	var/Bible_icon_state	// icon_state the chaplain has chosen for his bible
+	var/Bible_item_state	// item_state the chaplain has chosen for his bible
+	var/Bible_name			// name of the bible
+	var/Bible_deity_name
+
+	var/random_players = FALSE	// If set to nonzero, ALL players who latejoin or declare-ready join will have random appearances/genders
+
+	// TODO - Should this go here or in the job subsystem?
+	var/triai = FALSE // Global flag for Triumvirate AI being enabled
+
+	//station_explosion used to be a variable for every mob's hud. Which was a waste!
+	//Now we have a general cinematic centrally held within the gameticker....far more efficient!
+	var/obj/screen/cinematic = null
+
+// This global variable exists for legacy support so we don't have to rename every 'ticker' to 'SSticker' yet.
+var/global/datum/controller/subsystem/ticker/ticker
+/datum/controller/subsystem/ticker/PreInit()
+	global.ticker = src // TODO - Remove this! Change everything to point at SSticker intead
+	login_music = pick(\
+	/*'sound/music/halloween/skeletons.ogg',\
+	'sound/music/halloween/halloween.ogg',\
+	'sound/music/halloween/ghosts.ogg'*/
+	'sound/music/space.ogg',\
+	'sound/music/traitor.ogg',\
+	'sound/music/title2.ogg',\
+	'sound/music/clouds.s3m',\
+	'sound/music/space_oddity.ogg') //Ground Control to Major Tom, this song is cool, what's going on?
+
+/datum/controller/subsystem/ticker/Initialize()
+	pregame_timeleft = config.pregame_time
+	send2mainirc("Server lobby is loaded and open at byond://[config.serverurl ? config.serverurl : (config.server ? config.server : "[world.address]:[world.port]")]")
+	return ..()
+
+/datum/controller/subsystem/ticker/fire(resumed = FALSE)
+	switch(current_state)
+		if(GAME_STATE_INIT)
+			pregame_welcome()
+			current_state = GAME_STATE_PREGAME
+		if(GAME_STATE_PREGAME)
+			pregame_tick()
+		if(GAME_STATE_SETTING_UP)
+			setup_tick()
+		if(GAME_STATE_PLAYING)
+			playing_tick()
+		if(GAME_STATE_FINISHED)
+			post_game_tick()
+
+/datum/controller/subsystem/ticker/proc/pregame_welcome()
+	to_chat(world, "<B><FONT color='blue'>Welcome to the pregame lobby!</FONT></B>")
+	to_chat(world, "Please set up your character and select ready. The round will start in [pregame_timeleft] seconds.")

```

Comment:
Should these just be `to_world()`?
Could they also use a `<span class='filter_system'>` (Or some other filter tag)?

Reply:
These to_chats are just taken verbatim as they were when the code was moved around in this file.  But I don't mind changing them to to_world here, its good to do.

Reply:
I converted all the `to_chat(world, ...)` to   `to_world(...)` instead.   I also added wrapping <span> tags, except for the end of round declaration.  That one has particular formatting baked into it and probably needs a specialist's attention if you want to revamp it.   Its just end of round anyway.

Reply:
The `filter_` tags should be working now, so just applying one of those will work. I added them specifically to be able to precisely control which filter catches what message without tampering with the message styling